src/test/java/core/basesyntax/DiamondTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+
+public class DiamondTest {
+    private static final String ONE_STAR = "* \n";
+
+    private static final String EVEN_NUMBER_OF_STAR =
+                      "   * \n"
```

Comment:
do we want this extra space in the end? for the test case to be more reliable we can trim spaces in the end.

Reply:
done